models/Post.js

```diff
@@ -77,7 +77,7 @@ Post.add({
   leading_image_description: { label: '首圖圖說', type: String, require: false },
   brief: { label: '前言', type: Types.Html, wysiwyg: true, height: 150 },
   content: { label: '內文', type: Types.Html, wysiwyg: true, height: 400 },
-  categories: { label: '分類（舊）', type: Types.Relationship, ref: 'PostCategory', many: true },
+  categories: { label: '分類（舊）', type: Types.Relationship, ref: 'PostCategory', many: true, noedit: true },
```

Comment:
Question: why to we choose to set `categories` noedit instead of deleting it ?

Reply:
Because the data still exists in the database

Reply:
if we set categories noedit, would it be assigned with some default values or it would not be included in the new added post data ?